docs/userGuide/templates.md

```diff
@@ -0,0 +1,22 @@
+<variable name="title" id="title">Templates</variable>
+<frontmatter>
+  title: "User Guide: {{ title }}"
+  layout: userGuide
+</frontmatter>
+
+# Templates
+
+Markbind allows you to start off your new project with different templates.
+
+During initialization you can add a flag `--template <template-key>` to select a different template to initialize with. For example:
+
+```
+markbind init --template default
```

Comment:
Maybe use minimal as an example instead? default template shouldn't need to be specified.